lib/beaker-pe/install/pe_utils.rb

```diff
@@ -81,6 +81,68 @@ def sorted_hosts subset = hosts
           special_nodes + real_agents
         end
 
+        # If host or opts has the :use_puppet_ca_cert flag set, then push the master's
+        # ca cert onto the given host at /etc/puppetlabs/puppet/ssl/certs/ca.pem.
+        #
+        # This in turn allows +frictionless_agent_installer_cmd+ to generate
+        # an install which references the cert to verify the master when downloading
+        # resources.
+        def install_ca_cert_on(host, opts)
+          if host[:use_puppet_ca_cert] || opts[:use_puppet_ca_cert]
```

Comment:
I may be thinking of overly complicated use cases, but do we want to support the scenario where someone has set the top level option of `use_puppet_ca_cert = true`, but wants to disable it on only one host? Or would the answer there be "don't set top level, set the value on individual hosts"